app/enquiries/tests/test_views.py

```diff
@@ -68,12 +69,35 @@ def canned_enquiry():
         "project_success_date": date(2022, 2, 3),
     }
 
-
 class EnquiryViewTestCase(TestCase):
     def setUp(self):
+        """
+        Sets up test user to authenticate requests.
+        If you need to log out for a test remember to log bck in afterwards
+        """
         self.faker = Faker()
         self.client = Client()
 
+        self.CREDENTIALS = {
+            "username": "test",
+            "password": "user"
+        }
+        self.logged_in = False
+        UM = get_user_model()
```

Comment:
Would be good to use a more descriptive name than `UM` e.g. `user_model`:
```suggestion
        user_model = get_user_model()
```